yahoo_panoptes/framework/resources.py

```diff
@@ -832,18 +832,12 @@ def get_resources(self, query):
                                         resource_type=row[4], resource_id=row[5], resource_endpoint=row[6],
                                         resource_plugin=row[7])
 
-            try:
-                # Columns 8 & 9 contain the metadata keys and values respectively
-                if row[8] and row[9]:
-                    metadata_keys = row[8].split('|')
-                    metadata_values = row[9].split('|')
-                    for i in range(0, len(metadata_keys) - 1):
-                        resource.add_metadata(metadata_keys[i], metadata_values[i])
-            except Exception as e:
-                logger.error(
-                        'Either resource metadata key or value are not correct, skipping resource "%s": %s' % (
-                            row, str(e)))
-                continue
+            # Columns 8 & 9 contain the metadata keys and values respectively
```

Comment:
My guess is that this change was made to facilitate testing - however, this skipping of resources which can't be parsed is intentional. We need to revert these changes and the way to test the exception would be to insert a malformed resource directly into the mocked PanoptesResourceKVStore and check the count of fetched resources is one less than expected.

Reply:
Ok, I was ambivalent about removing the try-except, but chose to because I can't figure out how un-parseable metadata could be added to the resource because "add_metadata" has strict checks. Will add this back in on the significant chance I overlooked something.